nix/pkgs/kld.nix

```diff
@@ -51,7 +52,6 @@ craneLib.buildPackage {
       nativeBuildInputs = nativeBuildInputs ++ [ bitcoind cockroachdb ];
     };
     inherit cargoArtifacts;
-    COMMIT_SHA = lib.mkIf (self ? rev) self.rev;
```

Comment:
This cause a rebuild on every git push even if no code has changed. I rather would not have this as it increases the build times a lot.